analyzers/src/SonarAnalyzer.Common/Helpers/KnownType.cs

```diff
@@ -600,6 +605,13 @@ public sealed partial class KnownType
                 System_Runtime_CompilerServices_CallerFilePathAttribute,
                 System_Runtime_CompilerServices_CallerLineNumberAttribute,
                 System_Runtime_CompilerServices_CallerMemberNameAttribute);
+        public static readonly ImmutableArray<KnownType> DatabaseBaseQueryTypes =
+            ImmutableArray.Create(
+                System_Data_Entity_Infrastructure_DbQuery,
+                System_Data_Entity_Infrastructure_DbQuery_TResult,
+                Microsoft_EntityFrameworkCore_DbSet_TEntity,
+                System_Data_Linq_ITable,
+                System_Data_Entity_Core_Objects_ObjectQuery);
```

Comment:
Not fixable, but this misses [SqlQuery](https://learn.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.relationaldatabasefacadeextensions.sqlqueryraw?view=efcore-8.0#microsoft-entityframeworkcore-relationaldatabasefacadeextensions-sqlqueryraw-1(microsoft-entityframeworkcore-infrastructure-databasefacade-system-string-system-object())) and SqlQueryRaw.